solr/webapp/web/libs/ngtimeago.js

```diff
@@ -89,10 +89,9 @@ catalyst.filter('timeago', function() {
             days < 365 && substitute(strings.months, Math.round(days / 30), strings) ||
             years < 1.5 && substitute(strings.year, 1, strings) ||
             substitute(strings.years, Math.round(years), strings);
-            console.log(words+" "+prefix+suffix);
```

Comment:
Weirdly this was logging in the console periodically. It didn't seem to provide much value so removed it.